errors/rejected-promises.js

```diff
@@ -0,0 +1,6 @@
+exports.rejectedPromise404 = () => {
+  return Promise.reject({
+    status: 404,
+    msg: "Not found, please check the url and try again.",
```

Comment:
you may want to allow this to take an arg to change the error message for each rejection 😄